### PR TITLE
Modified README.md and  CONTRIBUTING.md 

### PR DESCRIPTION
**Pull request by MrPhnix**
Added winget command for git and instead of doing the ssh manually, use Github Desktop for easier git management.
Moved CONTRIBUTING to README.md for quicker access as it was not too long.

### CODE REVIEW

**Review by TPXP — COMMENTED** [review]
`CONTRIBUTING.md` has a special meaning to Github, it will be linked in the right bar of the project. I'd rather keep it and link it. It also allows us to offload the readme a little.

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types

We may add more files like CONTRIBUTING.md or move it to code_of_conduct.md though

File: README.md [hunk 1]
```diff
@@ -4,42 +4,58 @@ A Layton Fangame built with the [Unity Engine](https://unity.com).
 
 ## Getting started
 
-Start by installing the Unity Hub if you don't have it yet: https://unity.com/download
+### Install Github Desktop
```
Comment (TPXP) [inline README.md:7]: :thinking: Could we add a note saying something like "Already familiar with git? Just make sure lfs is installed" for people used to the git CLI?

File: README.md [hunk 2]
```diff
@@ -4,42 +4,58 @@ A Layton Fangame built with the [Unity Engine](https://unity.com).
...
+Start by installing the [Unity Hub](https://unity.com/download).<br>
 Then, install the Unity Editor with the Unity Hub. Make sure to pick version **2021.3.1f1**.
 
-If you don't have it yet, install [Git](https://git-scm.com/) on your machine too. Make
-sure you have [Git LFS too](https://git-lfs.github.com/). If you used the Git installer on
-Windows, it's installed by default.
+### Install Visual Studio Community 2022
 
-If this is your first time with Git or Github, [generate an SSH Key and add it to your Github account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+- You can choose any IDE, but for simplicity, we will install [Visual Studio](https://visualstudio.microsoft.com/).
```
Comment (TPXP) [inline README.md:19]:
```suggestion
- You can choose any IDE with C# support but, for simplicity, we will install [Visual Studio](https://visualstudio.microsoft.com/).
```

File: README.md [hunk 3]
```diff
@@ -4,42 +4,58 @@ A Layton Fangame built with the [Unity Engine](https://unity.com).
...
-Hi <user>! You've successfully authenticated, but GitHub does not provide shell access.
-Connection to github.com closed.
-```
+### Clone the repository
 
-Make sure you have setup Git LFS too:
-```
-git lfs install
-```
+If you don't have access to the main repository, fork it, create a new branch, work on it and request a pull request.
```
Comment (TPXP) [inline README.md:26]:
```suggestion
If you don't have access to the main repository, fork it, create a new branch, work on it and open a pull request.
```

File: README.md (hunk 1 above)
Comment (MrPhnix) [reply README.md:7]: Good idea! 👍

**Comment by MrPhnix** [comment]
I have modified the files according to your feedback. 😄

**Review by TPXP — APPROVED** [review]

**Pull request merged**